packages/webpack/src/loader.ts

```diff
@@ -20,14 +20,18 @@ interface LoaderContext {
   async: () => (err: unknown, result?: string) => void;
 }
 
+// Workaround to avoid consumers of the loader/webpack plugin having to add
+// virtual-resource-loader as a dependency when not it is not hoisted
```

Comment:
I think remove comment, potentially replace it with reasoning so it's clear:

```suggestion
// Resolve virtual-resource-loader dependency from current package
```

```suggestion
```